repose-aggregator/components/filters/header-normalization-filter/src/main/scala/org/openrepose/filters/headernormalization/HeaderNormalizationFilter.scala

```diff
@@ -71,22 +73,42 @@ class HeaderNormalizationFilter @Inject()(configurationService: ConfigurationSer
 
     val wrappedRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
 
-    config find { target =>
+    configRequest find { target =>
       // find the first "target" config element that matches this request (if any)
       target.url.matcher(wrappedRequest.getRequestURI).matches &&
```

Comment:
What I meant to say was the `Pattern` isn't cached, but it doesn't look like that is a big deal.